ushahidi/platform-client-mzima
# [Outreachy Task Submission]: Fix: email validation error in mobile profile settings

### Description:
This pull request addresses the issue: ushahidi/platform#4892, where an instant validation error occurs on the email input in the mobile profile settings. Currently, the error appears immediately without any user interaction triggering it, which can lead to confusion. 
### Testing Checklist:
1. Launch the mobile application using `npm run mobile:serve`.
2. Select a deployment or create a new one.
3. Log in to the application.
4. Click on the profile icon at the bottom navbar.
5. Within the popup, click on "Profile Information."
6. Within the popup, click on "Change Email."
7. Verify that the email input does not display an instant validation error message.
8. Ensure that the error message appears only when interacting with the input or attempting to submit the form without filling the required field. 

### Additional Context:
For more clarification, a video attachment has been provided on the issue [#4892](https://github.com/ushahidi/platform/issues/4892).
